packages/shared/src/components/history/ReadingHistoryList.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import {
+  HideReadHistory,
+  ReadHistoryInfiniteData,
+} from '../../hooks/useReadingHistory';
+import classed from '../../lib/classed';
+import {
+  isDateOnlyEqual,
+  getReadHistoryDateFormat,
+} from '../../lib/dateFormat';
+import ReadingHistoryItem from './ReadingHistoryItem';
+
+const DateTitle = classed('h2', 'typo-body text-theme-label-tertiary');
+
+const getDateGroup = (date: Date, isFirstLabel: boolean) => {
+  const label = getReadHistoryDateFormat(date);
+
+  return (
+    <DateTitle
+      key={label}
+      className={classNames('px-6 my-3', isFirstLabel && 'mt-0')}
```

Comment:
Just checking would `first:mt-0` not work here?

Reply:
To make the component predictable I applied the spacing in the below image from the title "Reading history", so consumers can have the decision how much margin should it be from where this will be used. And this actual component has a `my-3` margin

![Screen Shot 2021-11-16 at 5 06 54 PM](https://user-images.githubusercontent.com/13744167/141955590-a42dab56-6c4d-45ef-951e-b7ab8cd93d42.png)

Reply:
Ah, that should work!